content/blog/posts/2022-09-30-conference-basics.md

```diff
@@ -0,0 +1,67 @@
+---
+authorId: hannakim
+categories:
+  - devlife
+  - developertips
+  - community
+date: '2022-08-30'
```

Comment:
```suggestion
date: '2022-09-30'
```

Reply:
Ah good catch - I'll make a separate PR since this is already merged